tests/test_cli.py

```diff
@@ -1011,23 +1011,12 @@ def verify_noQC_citations_not_written(self, cmd, caplog):
     @pytest.mark.usefixtures("caplog")
     def test_noQC_citations_not_written(self, caplog):
         self.verify_noQC_citations_not_written(
-            "EscCol -d FakeModel -D 10 -L 10", caplog
+            "EscCol -d FakeModel -D 10 -L 1000", caplog
         )
 
     @pytest.mark.filterwarnings("ignore::stdpopsim.QCMissingWarning")
     @pytest.mark.usefixtures("caplog")
     def test_noQC_citations_not_written_verbose(self, caplog):
         self.verify_noQC_citations_not_written(
-            "-vv EscCol -d FakeModel -D 10 -L 10", caplog
+            "-vv EscCol -d FakeModel -D 10 -L 1000", caplog
         )
-
-
```

Comment:
why are you removing this test, btw?

Reply:
Normally, I wouldn't dare to remove a test 😉. But above @jeromekelleher suggested:
> We could probably just delete the test I'd imagine, it's not doing anything particularly useful anyway and is going to be harder and harder to make work as more things get added to the catalog.